src/SFA.DAS.Reservations.Domain/Reservations/Api/AccountReservationStatusRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using SFA.DAS.Reservations.Domain.Interfaces;
+
+namespace SFA.DAS.Reservations.Domain.Reservations.Api
+{
+    public class AccountReservationStatusRequest : IGetApiRequest
+    {
+        public AccountReservationStatusRequest(string baseUrl, long accountId)
+        {
+            BaseUrl = baseUrl;
+            AccountId = accountId;
+        }
+        
+        public long AccountId { get; }
+        public string BaseUrl { get; }
+        public string GetUrl => $"{BaseUrl}api/accounts/{AccountId}/status";
```

Comment:
Not sure if we have a test around this URL being correctly set that I could see in this PR